sklearn/cross_decomposition/_pls_.py

```diff
@@ -440,14 +440,14 @@ def inverse_transform(self, X, copy=True):
         This transformation will only be exact if n_components=n_features
         """
         check_is_fitted(self)
-        X = check_array(X, copy=copy, dtype=FLOAT_DTYPES)
+        X = check_array(X, copy=True, dtype=FLOAT_DTYPES)
```

Comment:
you don't need to copy here. Just leave the `copy` as default.

Reply:
Does it make sense to accept the copy parameter even if it will not affect the parameter X? I was thinking on removing copy from the function arg list  and only accept X (and making it copy=True x default)

Wether `copy` is true or false will not affect the execution or trace it leaves on the parameters.

Reply:
yes that's what I meant.